Fix IDE0130: NamespaceDoesNotMatchFolderStructure

```
X:\src\Microsoft.WSMan.Runtime\WSManSessionOption.cs(10,11): error IDE0130: Namespace "Microsoft.WSMan.Management" does not match folder structure, expected "Microsoft.WSMan.Runtime." [X:\src\Microsoft.WSMan.Runtime\Microsoft.WSMan.Runtime.csproj]
X:\src\Microsoft.PowerShell.CoreCLR.Eventing\DotNetCode\Eventing\Reader\CoTaskMemSafeHandle.cs(15,11): error IDE0130: Namespace "System.Diagnostics.Eventing.Reader" does not match folder structure, expected "Microsoft.PowerShell.CoreCLR.Eventing.DotNetCode.Eventing.Reader" [X:\src\Microsoft.PowerShell.CoreCLR.Eventing\Microsoft.PowerShell.CoreCLR.Eventing.csproj]
X:\src\Microsoft.PowerShell.CoreCLR.Eventing\DotNetCode\Eventing\EventDescriptor.cs(8,11): error IDE0130: Namespace "System.Diagnostics.Eventing" does not match folder structure, expected "Microsoft.PowerShell.CoreCLR.Eventing.DotNetCode.Eventing" [X:\src\Microsoft.PowerShell.CoreCLR.Eventing\Microsoft.PowerShell.CoreCLR.Eventing.csproj]
X:\src\Microsoft.PowerShell.CoreCLR.Eventing\DotNetCode\Eventing\Reader\CoTaskMemUnicodeSafeHandle.cs(15,11): error IDE0130: Namespace "System.Diagnostics.Eventing.Reader" does not match folder structure, expected "Microsoft.PowerShell.CoreCLR.Eventing.DotNetCode.Eventing.Reader" [X:\src\Microsoft.PowerShell.CoreCLR.Eventing\Microsoft.PowerShell.CoreCLR.Eventing.csproj]
X:\src\Microsoft.PowerShell.CoreCLR.Eventing\DotNetCode\Eventing\Reader\EventLogHandle.cs(15,11): error IDE0130: Namespace "System.Diagnostics.Eventing.Reader" does not match folder structure, expected "Microsoft.PowerShell.CoreCLR.Eventing.DotNetCode.Eventing.Reader" [X:\src\Microsoft.PowerShell.CoreCLR.Eventing\Microsoft.PowerShell.CoreCLR.Eventing.csproj]
X:\src\Microsoft.PowerShell.CoreCLR.Eventing\DotNetCode\Eventing\EventProvider.cs(11,11): error IDE0130: Namespace "System.Diagnostics.Eventing" does not match folder structure, expected "Microsoft.PowerShell.CoreCLR.Eventing.DotNetCode.Eventing" [X:\src\Microsoft.PowerShell.CoreCLR.Eventing\Microsoft.PowerShell.CoreCLR.Eventing.csproj]
X:\src\Microsoft.PowerShell.CoreCLR.Eventing\DotNetCode\Eventing\EventProviderTraceListener.cs(8,11): error IDE0130: Namespace "System.Diagnostics.Eventing" does not match folder structure, expected "Microsoft.PowerShell.CoreCLR.Eventing.DotNetCode.Eventing" [X:\src\Microsoft.PowerShell.CoreCLR.Eventing\Microsoft.PowerShell.CoreCLR.Eventing.csproj]
X:\src\Microsoft.PowerShell.CoreCLR.Eventing\DotNetCode\Eventing\Reader\NativeWrapper.cs(23,11): error IDE0130: Namespace "System.Diagnostics.Eventing.Reader" does not match folder structure, expected "Microsoft.PowerShell.CoreCLR.Eventing.DotNetCode.Eventing.Reader" [X:\src\Microsoft.PowerShell.CoreCLR.Eventing\Microsoft.PowerShell.CoreCLR.Eventing.csproj]
X:\src\Microsoft.PowerShell.CoreCLR.Eventing\DotNetCode\Eventing\UnsafeNativeMethods.cs(9,11): error IDE0130: Namespace "System.Diagnostics.Eventing" does not match folder structure, expected "Microsoft.PowerShell.CoreCLR.Eventing.DotNetCode.Eventing" [X:\src\Microsoft.PowerShell.CoreCLR.Eventing\Microsoft.PowerShell.CoreCLR.Eventing.csproj]
```

--- a/.globalconfig
+++ b/.globalconfig
@@ -1440,7 +1440,7 @@ dotnet_diagnostic.IDE0120.severity = warning
 
 # IDE0130: NamespaceDoesNotMatchFolderStructure
 # https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0130
-dotnet_diagnostic.IDE0130.severity = suggestion
+dotnet_diagnostic.IDE0130.severity = silent
 
 # IDE1001: AnalyzerChanged
 # https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide1001

